project/src/views/index/index.js

```diff
@@ -0,0 +1,50 @@
+window.onload = function () {
+    const divEl = document.querySelector('#app');
+    divEl.innerHTML = LoginPage();
+
+    const button = document.querySelector('.button');
+    const checkCredentials = function () {
+        const username = document.querySelector('#username').value;
```

Comment:
Why would you use `querySelector` here over other options such as, let's say `getElementById`?

Reply:
because querySelector works for anything, no matter if it is an id, class or element. So I simply write a querySelector and use whatever attribute I have defined for that certain element, or even the element itself.